<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorPantalla</class>
 <widget class="QDialog" name="EditorPantalla">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>1034</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="agregarGusano">
         <property name="icon">
          <iconset>
           <normaloff>../imagenes/wormwait.png</normaloff>../imagenes/wormwait.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="AgregarViga">
         <property name="icon">
          <iconset>
           <normaloff>../imagenes/viga.png</normaloff>../imagenes/viga.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="agregarVigaChica">
         <property name="icon">
          <iconset>
           <normaloff>../imagenes/viga.png</normaloff>../imagenes/viga.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="SelecionarHerramintas_Armas">
         <property name="text">
          <string>Selecionar Herramientas/Armas</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="quitar">
         <property name="text">
          <string>Quitar Elemento</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="mas">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="menos">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLineEdit" name="vidaGusano">
         <property name="text">
          <string>vida</string>
         </property>
         <property name="maxLength">
          <number>4</number>
         </property>
         <property name="cursorPosition">
          <number>4</number>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="ok">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="bazooka">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="granadaV">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="granadaR">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="banana">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="granadaS">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="dinamita">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="bate">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="aereo">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="teletransportador">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QPushButton" name="saveAs">
         <property name="text">
          <string>Save as</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="mortero">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Fondo De escenario</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cant">
         <property name="text">
          <string>Cantida De jugadores</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
